backend/wmg/api/v1.py

```diff
@@ -28,23 +26,30 @@ def primary_filter_dimensions():
 def query():
     request = connexion.request.json
     criteria = WmgQueryCriteria(**request["filter"])
-
     snapshot: WmgSnapshot = load_snapshot()
-    query = WmgQuery(snapshot)
-    expression_summary = query.expression_summary(criteria)
-    cell_counts = query.cell_counts(criteria)
-    dot_plot_matrix_df = build_dot_plot_matrix(expression_summary, cell_counts)
+    
+    if len(criteria.gene_ontology_term_ids)>0 and len(criteria.tissue_ontology_term_ids)>0:
```

Comment:
change WmgQueryCriteria to allow empty gene list so that you don't need to add a dummy gene variable to all your filter queries